src/main/java/frc/trigon/robot/poseestimation/poseestimator/PoseEstimator.java

```diff
@@ -128,11 +125,8 @@ private Matrix<N3, N1> averageDistanceToStdDevs(double averageDistance, int visi
     }
 
     private void putAprilTagsOnFieldWidget() {
-        final HashMap<Integer, Pose3d> tagsIdToPose = RobotPoseSourceConstants.TAG_ID_TO_POSE;
-
-        for (Integer currentID : tagsIdToPose.keySet()) {
-            final Pose2d tagPose = tagsIdToPose.get(currentID).toPose2d();
-            field.getObject("Tag " + currentID).setPose(tagPose);
+        for (Map.Entry<Integer, Pose3d> entry : RobotPoseSourceConstants.TAG_ID_TO_POSE.entrySet()) {
+            field.getObject("Tag " + entry.getKey()).setPose(entry.getValue().toPose2d());
```

Comment:
Extract to more variables here.

Reply:
The variable in the new commit is outside the loop to save resources btw, saw you doing it differently somewhere else